src/cdk/a11y/_index.scss

```
@@ -45,9 +47,9 @@
 /// to Microsoft browsers. Chrome can be included by checking for the `html[hc]`
 /// attribute, however Chrome handles high contrast differently.
 ///
-/// @param target Which kind of high contrast setting to target. Defaults to `active`, can be
-///    `white-on-black` or `black-on-white`.
-/// @param encapsulation Whether to emit styles for view encapsulation. Values are:
+/// @param {String} target Type of high contrast setting to target. Defaults to `active`, can be
+///     `white-on-black` or `black-on-white`.
+/// @param {string} encapsulation Whether to emit styles for view encapsulation. Values are:
```

Nit: should this be `String` (at least for consistency)

```suggestion
/// @param {String} encapsulation Whether to emit styles for view encapsulation. Values are:
```